msal_extensions/windows.py

```diff
@@ -81,7 +82,9 @@ def protect(self, message):
                 return result.raw()
             finally:
                 _LOCAL_FREE(result.pbData)
-        return b''
+
+        err_code = _GET_LAST_ERROR()
+        raise OSError(1, '', '', err_code)
```

Comment:
In one of your [earlier attempt](https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/19/commits/332f0bcb2b90c27b72df738f4219a6acca77c70c), you used `None` as the first parameter. Would that work? If so, I would suggest to stick with `None`. If it has to be an integer, then maybe choose a [non-existing integer](http://minirighi.sourceforge.net/html/errno_8h-source.html) such as `127`?

After that, the [automatic translation](https://docs.python.org/3/library/exceptions.html#OSError.winerror) would kick in and translate it into a who-knows-what value, therefore the end result would probably still be the same, determined by that auto-translation mechanism (and the weather). But it would feel semantically more appropriate to NOT arbitrarily choose an potentially misleading (albeit admittedly close enough) error code `1`.

Reply:
Fixed in 15469c3310e4cbf66827a54f68efdf411be35345